biogeochem/EDPatchDynamicsMod.F90

```diff
@@ -1061,19 +1134,27 @@ subroutine mortality_litter_fluxes(currentSite, cp_target, new_patch_target, pat
     currentCohort => currentPatch%shortest
     do while(associated(currentCohort))       
        p = currentCohort%pft
+       
+       sapw_c   = currentCohort%prt%GetState(sapw_organ, all_carbon_species)
+       struct_c = currentCohort%prt%GetState(struct_organ, all_carbon_species)
+       leaf_c   = currentCohort%prt%GetState(leaf_organ, all_carbon_species)
+       fnrt_c   = currentCohort%prt%GetState(fnrt_organ, all_carbon_species)
+       store_c  = currentCohort%prt%GetState(store_organ, all_carbon_species)
 
           if(currentCohort%canopy_layer == 1)then         
              !currentCohort%dmort = mortality_rates(currentCohort) 
              !the disturbance calculations are done with the previous n, c_area and d_mort. So it's probably &
              !not right to recalcualte dmort here.
              canopy_dead = currentCohort%n * min(1.0_r8,currentCohort%dmort * hlm_freq_day * fates_mortality_disturbance_fraction)
 
+
+
              canopy_mortality_woody_litter(p)= canopy_mortality_woody_litter(p) + &
-                  canopy_dead*(currentCohort%bdead+currentCohort%bsw)
+                  canopy_dead*(struct_c + sapw_c)
              canopy_mortality_leaf_litter(p) = canopy_mortality_leaf_litter(p) + &
-                  canopy_dead*(currentCohort%bl)
+                  canopy_dead*leaf_c
              canopy_mortality_root_litter(p) = canopy_mortality_root_litter(p) + &
-                  canopy_dead*(currentCohort%br+currentCohort%bstore)
+                  canopy_dead*(fnrt_c + store_c)
```

Comment:
@rgknox FYI use of "dead" here - justing pointing you to it

Reply:
thanks @serbinsh , in this case, dead really refers to dead trees, specifically the number density of dead trees